msteams-platform/includes/tabs/content-security-policy-headers.md

```diff
@@ -1,12 +1,13 @@
+| **Microsoft 365 app host** | **frame-ancestor permission** |
+|--|--|
+| *All hosts* (New) | `*.cloud.microsoft` |
+| Teams | `teams.microsoft.com`, `*.teams.microsoft.com` |
+| Microsoft 365 app | `*.microsoft365.com`, `*.office.com`,  |
+| Outlook | `outlook.office.com`, `outlook.office365.com`, `outlook-sdf.office.com`, `outlook-sdf.office365.com` |
+
 > [!WARNING]
-> Microsoft's cloud services, including web versions of Teams (*teams.microsoft.com*), Outlook (*outlook.com*), and Microsoft 365 (*microsoft365.com*) domains are migrating to the *cloud.microsoft* domain. Perform the following steps before June 2024 to ensure your app continues to render on the Teams web client:
+> Microsoft's cloud services, including web versions of Teams, Outlook  and Microsoft 365 domains are migrating to the *cloud.microsoft* domain. Perform the following steps before June 2024 to ensure your app continues to render on supported Microsoft 365 web client hosts:
 >
-> 1. Update TeamsJS SDK to v.2.19.0 or higher. For more information about the latest release of TeamsJS SDK, see [Microsoft Teams JavaScript client library](https://www.npmjs.com/package/@microsoft/teams-js).
+> 1. Update TeamsJS library to v.2.19.0 or higher. For more information about the latest release of TeamsJS, see [Microsoft Teams JavaScript client library](https://www.npmjs.com/package/@microsoft/teams-js).
 >
-> 2. Update your [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) headers in your Teams app to allow your app to access the ***teams.cloud.microsoft*** domain. If your Teams app extends across Outlook and Microsoft 365, ensure you allow your app to access ***teams.cloud.microsoft***, ***outlook.cloud.microsoft***, and ***m365.cloud.microsoft*** domains.
-
-| **Microsoft 365 app host** | **frame-ancestor permission** |
-|--|--|
-| Teams | `teams.microsoft.com`, `*.teams.microsoft.com`, `teams.cloud.microsoft` |
-| Microsoft 365 app | `*.microsoft365.com`, `*.office.com`, `m365.cloud.microsoft` |
-| Outlook | `outlook.live.com`, `outlook.office.com`, `outlook.office365.com`, `outlook-sdf.office.com`, `outlook-sdf.office365.com`, `outlook.cloud.microsoft` |
+> 2. Update your [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) headers in your Teams app to allow your app to access the ****.cloud.microsoft*** domain. This ensures your app continues to work across existing and future Microsoft 365 host applications and minimizes the need for subsequent changes.
```

Comment:
"to access" is ambiguous here, and technically it's the other way around-- the app is allowing itself to be iframed by Microsoft services. I'd recommend being more specific/prescriptive, something like ".. to include the `*.cloud.microsoft` domain in the `frame-ancestors` directive.".

Reply:
Thanks @aosolis, good call. How about this: 

Update your Teams app's [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) header [frame-ancestors](https://developer.mozilla.org/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors) directive to allow your app to be hosted by the *.cloud.microsoft domain. This ensures that your app continues to work across existing and future Microsoft 365 host applications and minimizes the need for subsequent changes.

Reply:
Looks good to me!